spec/factories/targets.rb

```diff
@@ -6,5 +6,6 @@
     target_count 100
     unit Target.units[:submissions]
     conference
+    campaign
   end
```

Comment:
Basically I'm with @differentreality, you shouldn't add fields / attributes to the factory which are not necessary to get a valid factory. However, cleaner code is also a valid point. How about [inheritance](https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#inheritance)?